wrftladj/solve_em_ad.F

```diff
@@ -3015,8 +3014,8 @@ SUBROUTINE solve_em_ad ( grid , config_flags  &
                                    kde,ims,ime,jms,jme,kms,kme,ips,ipe,&
                                    jps,jpe,kps,kpe,grid%i_start(ij),   &
                                    grid%i_end(ij),grid%j_start(ij),    &
-                                   grid%j_end(ij),k_start,k_end)
-!             IF(is .ne. P_QNN)THEN
+                                   grid%j_end(ij),k_start,k_end)        ! for ntu3m
+!ntu3m             IF(is .ne. P_QNN)THEN
```

Comment:
@tzuchin12 
It looks like this code is purposefully commented out. Place your "!ntu3m" identifier AFTER the Fortran statement. Also at line 3039.

Reply:
Corrections have been made.